lib/src/bluetooth_device.dart

```diff
@@ -117,12 +122,12 @@ class BluetoothDevice {
 
   @override
   bool operator ==(Object other) {
-    if (!(other is BluetoothDevice)) {
+    if (other is! BluetoothDevice) {
```

Comment:
Didn't know this existed

Reply:
There is now a lint for that. The IDE proposed the fix